department-of-veterans-affairs/abd-vro
# MCP-2281: Persist assessment result in V2

## What was the problem?

A claim is persisted for MAS processing but assessment results where not attach

Associated tickets or Slack threads:

- [MCP-2281](https://amida.atlassian.net/browse/MCP-2281)

## How does this fix it?[^1]

Attach assessment results to the claim after calling the "health evidence API"

## How to test this PR
- Run automatedClaim with defaults
- Query the claims via Swagger. The peristed claim has evidence summary and assessment results


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
